packages/widget-chat/src/components/icon/index.js

```diff
@@ -0,0 +1,27 @@
+import React, {PropTypes} from 'react';
+
+import classNames from 'classnames';
+
+import styles from './styles.css';
+
+function Icon(props) {
+  let iconCode = ``;
+  if (props.type === `delete`) {
```

Comment:
Maybe instead we should use what is setup  and just apply css classes instead of text?